frontend/packages/console-app/src/components/network-policies/network-policy-form.tsx

```diff
@@ -166,6 +179,35 @@ export const NetworkPolicyForm: React.FunctionComponent<NetworkPolicyFormProps>
 
   return (
     <Form onSubmit={save} className="co-create-networkpolicy">
+      {showSDNAlert &&
+        networkFeaturesLoaded &&
+        networkFeatures?.PolicyEgress === undefined &&
+        networkFeatures?.PolicyPeerIPBlockExceptions === undefined && (
+          <Alert
+            variant="info"
+            title={<p>{t('public~When using the OpenShift SDN cluster network provider:')}</p>}
```

Comment:
I *think* the alert `title` gets a heading element, in which case you wouldn't want to use `p`. Does this change the padding for the layout?